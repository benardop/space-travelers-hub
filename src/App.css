@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');

:root {
  --color-black: #212529;
  --color-blue: #157af6;
  --color-greenishblue: #42a2b9;
  --color-white: #fff;
}

.App {
  margin: 8rem 0 5rem 0;
  padding: 0;
  box-sizing: border-box;
  font-family: 'Poppins', sans-serif;
}

html {
  scroll-behavior: smooth;
}

