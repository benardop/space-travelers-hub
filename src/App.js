import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from './redux/configureStore';
import 'bootstrap/dist/css/bootstrap.min.css';
import './App.css';
import Navbar from './components/Navbar';
import Rockets from './components/Rockets';
import Missions from './components/Missions';
import MyProfile from './components/MyProfile';

function App() {
  return (
    <Provider store={store}>
      <Router>
        <Navbar />
        <Routes>
          <Route path="/" element={<Rockets />} />
          <Route path="/missions" element={<Missions />} />
          <Route path="/myprofile" element={<MyProfile />} />
        </Routes>
      </Router>
    </Provider>
  );
}

export default App;
