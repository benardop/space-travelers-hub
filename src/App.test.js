import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import store from './redux/configureStore';
import App from './App';

const MockApp = () => (
  <Provider store={store}>
    <BrowserRouter>
      <App />
    </BrowserRouter>
  </Provider>
);

test('renders correctly', () => {
  const tree = renderer
    .create(<MockApp />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
