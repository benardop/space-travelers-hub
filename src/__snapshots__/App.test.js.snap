// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <header>
    <div
      className="logo"
    >
      <img
        alt="logo"
        className="space-logo"
        src="planet.png"
      />
      <span
        className="title-logo"
      >
        Space Travelers'  Hub
      </span>
    </div>
    <nav>
      <ul>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={
            Object {
              "fontWeight": "bold",
              "textDecoration": "underline",
            }
          }
        >
          <span
            className="rockets"
          >
            Rockets
          </span>
        </a>
        <a
          className=""
          href="/missions"
          onClick={[Function]}
          style={
            Object {
              "fontWeight": "normal",
              "textDecoration": "none",
            }
          }
        >
          <span
            className="missions"
          >
            Missions
          </span>
        </a>
        <a
          className=""
          href="/myprofile"
          onClick={[Function]}
          style={
            Object {
              "fontWeight": "normal",
              "textDecoration": "none",
            }
          }
        >
          <span
            className="myprofile"
          >
            My Profile
          </span>
        </a>
      </ul>
    </nav>
  </header>
  <section
    className="footer"
  >
    Copyright ©️ 2022
    <a
      href="https://github.com/benardop"
    >
      benardop
    </a>
    <a
      href="https://github.com/mirwaisfarahi"
    >
      mirwaisfarahi
    </a>
    <img
      alt="github-logo"
      className="footerImg"
      src="github.svg"
    />
  </section>
</div>
`;
